heron/tools/cli/src/python/main.py

```diff
@@ -234,14 +234,16 @@ def main():
 
   start = time.time()
   resp = run(command, parser, command_line_args, unknown_args)
-  response.render(resp)
+  if command not in ('help', 'version'):
+    response.render(resp)
   end = time.time()
 
   if command not in ('help', 'version'):
     sys.stdout.flush()
     Log.info('Elapsed time: %.3fs.', (end - start))
-
-  return 0 if response.isAllSuccessful(resp) else 1
+    return 0 if response.isAllSuccessful(resp) else 1
+  else:
+    return 0 if resp else 1
```

Comment:
not the case for `help` and `version` subcommands. I feel it is OK to let them return a boolean as always, instead of making them to return a response object